Configure

```diff
@@ -940,6 +929,17 @@ if ($target eq "HASH") {
 print "Configuring OpenSSL version $config{version} ($config{version_num}) ";
 print "for $target\n";
 
+if (scalar(@seed_sources) == 0) {
+    print "Using implicit seed configuration\n";
```

Comment:
I'm not sure if "implicit" is the right word.  Maybe os-specific?

Reply:
Sure